src/allmydata/test/test_client.py

```diff
@@ -71,7 +71,7 @@ def test_unreadable_config(self):
         old_mode = os.stat(fn).st_mode
         os.chmod(fn, 0)
         try:
-            e = self.assertRaises(EnvironmentError, read_config, basedir, "client.port")
+            e = self.assertRaises(EnvironmentError, read_config, basedir, "client.port", _valid_config_sections=client._valid_config_sections)
```

Comment:
maybe would be nice to break these expressions over multiple lines now

Reply:
haha, I was GOING to do that, then thought "whitespace PLUS other changes, maybe less readable for the diff" ;)